metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi_utils.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+
+def formulate_description(
+    display_name: Optional[str], description: Optional[str]
+) -> str:
+    if display_name and description:
+        return f"{display_name}\n-----\n{description}"
+
+    if display_name:
+        return display_name
+
+    return ""
```

Comment:
Can we return None from here ? and make return type as optional[str]

Reply:
The `DashboardInfoClass` expects description to be `str` and not `Optional[str]`, if I change the return value to be optional, then all the descriptions change to `description=formulate_description(....) or ""` which doesn't look too good to my eye. I'd like to keep the behavior as is just because it looks cleaner, what do you think?

Reply:
If that is the case then use str() , it returns empty string. and this is the default value for description in DashboardInfoClass

Reply:
I'm not sure whether I fixed it with my change, but I tried! :)